templates/types/streaming/nextjs/README-template.md

```diff
@@ -26,6 +26,39 @@ You can start editing the page by modifying `app/page.tsx`. The page auto-update
 
 This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
 
+## Using docker
+
+1. Build an image for Next app:
+
+```
+docker build -t <your_app_image_name> .
+```
+
+2. Start the app:
+
+- Generate the embedding for `./data` folder if it exists - otherwise, skip this step:
```

Comment:
```suggestion
2. Generate embeddings:

Parse the data and generate the vector embeddings if the `./data` folder exists - otherwise, skip this step:
```